helm/chaos-mesh/templates/controller-manager-deployment.yaml

```diff
@@ -121,10 +125,6 @@ spec:
             value: /etc/chaos-daemon/cert/tls.key
           - name: CHAOS_MESH_CA_CERT
             value: /etc/chaos-daemon/cert/ca.crt
-          - name: QPS
```

Comment:
not sure why QPS and BURST was in the `Values.chaosDaemon.mtls.enabled` section in first place